packages/cli/package.json

```diff
@@ -25,16 +25,16 @@
     "docs": "ZAPIER_BASE_ENDPOINT='' node scripts/docs.js",
     "preversion": "git pull && yarn validate",
     "prepack": "oclif manifest",
-    "postpack": "rm -f oclif.manifest.json",
+    "postpack": "rimraf oclif.manifest.json",
     "precommit": "yarn docs && git add docs",
     "version": "yarn docs && git add docs/*",
     "postversion": "git push && git push --tags",
     "lint": "eslint src",
     "lint:fix": "eslint --fix src",
-    "test": "cross-env NODE_ENV=test mocha -t 50s --recursive src/tests --exit",
-    "test:debug": "cross-env NODE_ENV=test node inspect ../../node_modules/.bin/mocha -t 50s --recursive src/tests --exit",
-    "smoke-test": "cross-env NODE_ENV=test mocha -t 2m --recursive src/smoke-tests --exit",
-    "smoke-test:debug": "cross-env NODE_ENV=test node inspect ../../node_modules/.bin/mocha -t 2m --recursive src/smoke-tests --exit",
+    "test": "cross-env NODE_ENV=test mocha -t 200s --recursive src/tests --exit",
+    "test:debug": "cross-env NODE_ENV=test node inspect ../../node_modules/.bin/mocha -t 200s --recursive src/tests --exit",
+    "smoke-test": "cross-env NODE_ENV=test mocha -t 6m --recursive src/smoke-tests --exit",
+    "smoke-test:debug": "cross-env NODE_ENV=test node inspect ../../node_modules/.bin/mocha -t 6m --recursive src/smoke-tests --exit",
```

Comment:
GitHub Actions's Windows runners are [terribly slow](https://github.com/actions/runner-images/issues/7320), so I increased the time limits to avoid timeouts.